core/core.go

```diff
@@ -49,6 +51,11 @@ func NewCore(storePath string, nodeId bundle.EndpointID, inspectAllBundles bool,
 	c.InspectAllBundles = inspectAllBundles
 	c.NodeId = nodeId
 
+	c.CLAdapters = make(map[CLAType][]bundle.EndpointID)
```

Comment:
This approach seems to be very strict. If one adds some CLA, it also needs to be added here, right?

Reply:
I could move the whole thing to the cla Manager and do it all in the `Register` method?